### ISSUE TITLE
Should we increase the screen by two (or x) tiles wide?

### ISSUE THREAD

**Issue by Zth--** [d3athrow/vgstation13#17742]
I personally don't like our game screen being a perfect 15x15 square, I think we could use our gamescreen better by having a 17x15 or even a 18x15 rectangle. Here are some screenshots for reference:

current 15x15: http://i.imgur.com/HAuvMub.png
16x15: http://i.imgur.com/uqOsaAs.png
17x15: http://i.imgur.com/BeUrUnK.png (I think this one is the best option)
18x15: http://i.imgur.com/FkV6eh0.png

Note that I'm resizing the chat to perfectly fit the screen.

Codewise this is as simple as editing the next line and addressing some runtimes.
https://github.com/vgstation-coders/vgstation13/blob/882b9a9bd7ef8db051980b94431d1d7c948e253a/code/world.dm#L6

**Comment by ajc1g15** [comment]
As nice it it would, or could look. This will actually affect game balance somewhat, by which I mean a lot.

Think on it, currently being more than 7 tiles away from someone means they can't see you, unless they have a scope or some vision augment. This helps with getting away with shit and stealth and the like.

With these proposed changes, that becomes a fair bit more difficult, so horizontal stealth is more difficult, while vertical stealth is easier and thus preferable.

Also it makes no sense in universe either. Why can people see further in some arbitrary direction than others? The screen limit basically acts as a hard limit on the distance one can see so it can be assumed that the average view distance for most entities is 7.5 tiles, or about 15 meters.

Several overlays would be broken by this too, like we used to have an issue with the sleeping screen overlay regarding larger than usual view distances

So I'd say no to this change, having the view being a square seems suitable for balancing vertical vs horizontal view distance. Most realistically view distance should be a 7.5 radius circle, but that would be a bitch to implement all things considered. So a square is the most feasible balance

Tl;dr: Changing this would affect several of the core game mechanics. So while it would be "easy" to implement, the effect it would have go far beyond simple aesthetics

**Comment by Aurx** [comment]
The only sane reason to do this would be to black out the left and right columns and use them exclusively for UI elements.  
There's literally no justification for having a greater lateral view range than vertical.

**Comment by ajc1g15** [comment]
If I remember the UI already can be placed outside of proper game space if you resize the window in the right way.

**Comment by Zth--** [comment]
Universe doesn't make sense already, so I don't think realism has to be justified in this particular case (why can you see in every direction at the same time? do you have eyes on your back?).

Overlays is a fair point, but that can be addressed.

I feel like we're not using our space wisely, I'm only using half my monitor to play a game, while having the other half being a fuckhuge chat or having two black columns of unused space. I like the idea of having UI elements moved to the sides.

I'm not so concerned in having to justify some things as long as the game feels more pleasant to play with.

**Comment by Reviire** [comment]
What sense would it make to have it un-even? I'd like if the game screen size was increased in general, though. It feels too small for how fast we move.

**Comment by ghost** [comment]
Maybe we can finally move the spell bar outside the fucking game world

**Comment by Exxion** [comment]
Discussion dead
Reopen if revived

And yes, it is possible to use the additional horizontal space for HUD only
